packages/smarthr-ui/src/components/Experimental/SideMenu/SideMenuGroup.tsx

```diff
@@ -35,26 +36,34 @@ export const SideMenuGroup = <TitleElement extends ElementType = 'p'>({
   children,
   className,
 }: Props<TitleElement>) => {
-  const { wrapperStyle, listStyle, groupTitleStyle } = useMemo(() => {
-    const { wrapper, list, groupTitle } = sideMenuGroup()
+  const classNames = useMemo(() => {
+    const { wrapper, list, groupTitle } = classNameGenerator()
+
     return {
-      wrapperStyle: wrapper({ className }),
-      listStyle: list(),
-      groupTitleStyle: groupTitle(),
+      wrapper: wrapper({ className }),
+      list: list(),
+      groupTitle: groupTitle(),
     }
   }, [className])
 
-  const TitleComponent = titleElementAs ?? 'p'
   const ListComponent = listElementAs ?? 'ul'
 
   return (
-    <li className={wrapperStyle}>
-      <TitleComponent>
-        <Text color="TEXT_BLACK" leading="TIGHT" size="S" weight="bold" className={groupTitleStyle}>
-          {title}
-        </Text>
-      </TitleComponent>
-      <ListComponent className={listStyle}>{children}</ListComponent>
+    <li className={classNames.wrapper}>
+      <GroupTitleText titleElementAs={titleElementAs} className={classNames.groupTitle}>
+        {title}
+      </GroupTitleText>
+      <ListComponent className={classNames.list}>{children}</ListComponent>
     </li>
   )
 }
+
+const GroupTitleText = memo<PropsWithChildren<{ titleElementAs?: ElementType; className: string }>>(
+  ({ titleElementAs: Component = 'p', children, className }) => (
```

Comment:
imo: Wrapper など意味のある名前にしてもよさそう

```suggestion
  ({ titleElementAs: Wrapper = 'p', children, className }) => (
```

Reply:
あー、確かに。
ただ他コンポーネントの実装で↑のようなパターンの場合、ほとんどがComponentという変数にしているのでそれに習った感じです。

個人的にはここは範囲も小さいし、他に習っておくほうが読みやすそうな印象なんですがどうでしょう？

Reply:
スコープも狭いので強いこだわりはないですが、Component よりは Wrapper の方が良いとは思います!
他と合わせるメリットよりもこの場での役割が明確な方が良いと思います。

Reply:
取り込みますか！